packages/api/src/hooks/useTradingPlatformDeposit.ts

```diff
@@ -0,0 +1,21 @@
+import { useCallback } from 'react';
+import useMutation from '../useMutation';
+
+type TPayload = Parameters<ReturnType<typeof useMutation<'trading_platform_deposit'>>['mutate']>[0]['payload'];
+
+/** A custom hook for top-up of Other CFD Accounts */
+const useTradingPlatformDeposit = () => {
```

Comment:
Better to name this like OtherCFDPlatformDeposit or something better

Reply:
We should use it as an entity based name right instead of generalize one

Reply:
Thought it would be better to name it according to the endpoints we're calling, i.e. 
`landing_company` -> `useLandingCompany`
`trading_platform_investor_password_change` -> `useTradingPlatformInvestorPasswordChange`
`verify_email` -> `useVerifyEmail`

Reply:
But this is named from the API. The next person working on this wont know what it is. Plus TradingPlatformDeposit can also include mt5 if unclear. We have `useCreateOtherCFDAccount`.

Reply:
If we name it according to the api, there's no difference in WS. or calling useMutation in the component itself

Reply:
Makes sense so we can rename it to useOtherCFDDeposit